src/main/java/com/iexec/core/replicate/ReplicateSupplyService.java

```diff
@@ -104,6 +111,7 @@ public ReplicateSupplyService(ReplicatesService replicatesService,
                 if (enclaveChallenge.isEmpty()) continue;
 
                 replicatesService.addNewReplicate(chainTaskId, walletAddress);
+                taskService.setTaskAccessedForNewReplicate(chainTaskId, false);//release task when replicate is created
```

Comment:
We better call the method releaseXXX explicitly instead of setXXX.

Reply:
Thanks!